ui/src/app/cohort-search/list-overview/list-overview.component.tsx

```diff
@@ -0,0 +1,426 @@
+      ];
+      return <React.Fragment>
+        <div>
+          <div style={styles.overviewHeader}>
+            <div style={{width: '100%'}}>
+              {!!cohort && <React.Fragment>
+                <Menu appendTo={document.body}
+                  model={items} popup={true} ref={el => this.dropdown = el} />
+                <Button type='primary' style={styles.saveButton}
+                  onClick={(event) => this.dropdown.toggle(event)}
+                  disabled={loading || saving || this.hasErrors}>
+                  SAVE COHORT <ClrIcon shape='caret down' />
+                </Button>
+              </React.Fragment>}
+              {!cohort && <Button type='primary'
+                onClick={() => this.setState({saveModal: true})}
+                style={styles.saveButton}
+                disabled={loading || this.hasErrors}>CREATE COHORT</Button>}
+              <TooltipTrigger content={<div>Export to notebook</div>}>
+                <Clickable style={{...styles.actionIcon, ...styles.disabled}}
+                  onClick={() => this.navigateTo('notebook')} disabled>
+                  <ClrIcon shape='export' className='is-solid' size={30} />
+                </Clickable>
+              </TooltipTrigger>
+              <TooltipTrigger content={<div>Delete cohort</div>}>
+                <Clickable style={{...styles.actionIcon, ...(disableIcon ? styles.disabled : {})}}
+                  onClick={() => this.setState({deleting: true})}>
+                  <ClrIcon shape='trash' className='is-solid' size={30} />
+                </Clickable>
+              </TooltipTrigger>
+              <TooltipTrigger content={<div>Review participant level data</div>}>
+                <Clickable style={{...styles.actionIcon, ...(disableIcon ? styles.disabled : {})}}
+                  onClick={() => this.navigateTo('review')}>
+                  <ClrIcon shape='copy' className='is-solid' size={30} />
+                </Clickable>
+              </TooltipTrigger>
+            </div>
+            <h2 style={styles.totalCount}>
+              Total Count: &nbsp;
+              {loading && !this.hasTemporalError && <Spinner size={18} />}
+              {!loading && !this.hasErrors && total !== undefined &&
+                <span>{total.toLocaleString()}</span>
+              }
+              {this.hasErrors && <span>
+                -- <TooltipTrigger content={this.hasTemporalError ?
+                    'Please complete criteria selections before saving temporal relationship.' :
+                    `All criteria are suppressed. Un-suppress criteria to update the total count
+                     based on the visible criteria.`}>
+                  <ClrIcon style={{color: '#F57600'}} shape='warning-standard' size={18} />
+                </TooltipTrigger>
+              </span>}
+            </h2>
+          </div>
+          {error && !this.hasErrors && <div style={styles.totalError}>
```

Comment:
see my naming comment where state is defined.  the whole "error && not error" phrasing is a bit confusing to me; it would be nicer if it were clear what kind of error this was

Reply:
Done.